Userland/Libraries/LibGfx/ImageFormats/JBIG2Loader.cpp

```diff
@@ -768,6 +768,10 @@ static ErrorOr<void> warn_about_multiple_pages(JBIG2LoadingContext& context)
     return {};
 }
 
+struct AdaptiveTemplatePixel {
+    i8 x, y;
```

Comment:
I know this already existed, but these should be one line each:
https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md#other-punctuation
and initialized as well.